Personas/procurement-parker.md

```diff
@@ -0,0 +1,24 @@
+# Procurement Parker
+
+## Description
+Parker is responsible for evaluating large procurement projects for government departments. They've taken the DITAP course, so they are familiar with agile and how that plays into procurement. They aren't very familiar with accessibility, but they know they have to do more than insert a line about 508 into an RFP.
+
+Parker is hard of hearing and has trouble communicating with people over the phone. They prefer communications over email where they make sure they understand. 
```

Comment:
Nitpick. I think the second sentence was trying to convey a thought but got cut off.